lib/mix/tasks/phoenix.new.ex

```diff
@@ -33,6 +34,12 @@ defmodule Mix.Tasks.Phoenix.New do
 
     copy_from template_dir, path, application_name, &EEx.eval_file(&1, binding)
     copy_from static_dir, Path.join(path, "priv/static"), application_name, &File.read!(&1)
```

Comment:
We should not copy anything to `priv/static` anymore and we should rather .gitignore it.